sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -107,6 +107,10 @@ public String getVaultUrl() {
         return vaultUrl;
     }
 
+    Duration getPollDuration() {
+        return Duration.ofSeconds(1);
```

Comment:
Can we create a constant and name it as `DEFAULT_POLL_DURATION`?

Reply:
Done